scylla-gdb.py

```diff
@@ -1295,6 +1296,11 @@ def _make_symbol_matchers(symbol_specs):
     def _make_symbol_matcher(symbol_spec):
         unmangled_prefix = 'vtable for {}'.format('::'.join(symbol_spec))
         def matches_symbol(name):
+            name = name.strip()
+            if name.endswith(")") or (") [clone ." in name and name.endswith("]")):
```

Comment:
Why is a name ending with `)` enough to qualify it as a coroutine? What suffix are you are looking for? The commit message suggests `[clone .resume]`, where does the `)` come in here?

Reply:
`[clone .resume]` is one possibility (those probably correspond to `co_await` points in the middle of coroutines), but entries without that (i.e. just coroutine function names) also happen

Reply:
Ok, we can start with this and refine once we see some broken traces.

Reply:
I'd consider a regular expression for the right part of the expression above. like
`re.search(r'\) \[clone \.\w+\]$', name)`
(can be optimized by one time re.compile of the regexp above)